src/test_driver/mbed/unit_tests/main/main.cpp

```diff
@@ -63,8 +62,12 @@ int main()
 
     ChipLogProgress(NotSpecified, "Mbed unit-tests application run");
 
-    int status = RunRegisteredUnitTests();
-    ChipLogProgress(NotSpecified, "CHIP test status: %d", status);
+    int status = 0;
```

Comment:
status should probably not be 0, and instead we could log an error and then return non-zero.